### PR TITLE
Update documentation with latest changes to MRO shared lib

### LINKED ISSUE #92: Update documentation with latest changes to MRO shared lib
Update documentation to reflect changes made in:
- https://github.com/opendevstack/ods-mro-jenkins-shared-library/issues/10
- https://github.com/opendevstack/ods-mro-jenkins-shared-library/issues/22

### PR DESCRIPTION
**Pull request by metmajer**
Closes #92.

### CODE REVIEW

**Comment by metmajer** [comment]
Requires approval of https://github.com/opendevstack/ods-mro-jenkins-shared-library/pull/23.

**Comment by clemensutschig** [comment]
@metmajer - cool ... now you need to seed release... with the right types.. ;)

File: release-manager/files/README.md [hunk 1]
```diff
@@ -22,13 +22,44 @@ repositories:
...
+    name: my-repo-B
+    branch: master
+    type: ods
+  - id: C
+    type: ods
+```
+
+#### Repository Type: ods
+
+This type designates ODS components designed for _code development_. Examples include repositories based on quickstarters whose names start with `fe-` or `be-`, for _frontend_ and _backend_, respectively. This is the default.
```
Comment (clemensutschig) [inline release-manager/files/README.md]: Add  ds- ... Those are code as well

Comment (metmajer) [reply release-manager/files/README.md]: Resolved via https://github.com/opendevstack/ods-quickstarters/pull/93/files#diff-84a836f08dec6674fd9246400c0c0d78R51-R53.

**Pull request merged**